src/testUtils/java/uk/gov/hmcts/reform/wataskmonitor/config/GivensBuilder.java

```diff
@@ -0,0 +1,259 @@
+                         RestApiActions restApiActions,
+                         AuthorizationHeadersProvider authorizationHeadersProvider,
+                         CoreCaseDataApi coreCaseDataApi,
+                         DocumentManagementFiles documentManagementFiles
+    ) {
+        this.camundaApiActions = camundaApiActions;
+        this.restApiActions = restApiActions;
+        this.authorizationHeadersProvider = authorizationHeadersProvider;
+        this.coreCaseDataApi = coreCaseDataApi;
+        this.documentManagementFiles = documentManagementFiles;
+
+    }
+
+    public String createCcdCase() {
+        Headers headers = authorizationHeadersProvider.getLawFirmAuthorization();
+        String userToken = headers.getValue(AUTHORIZATION);
+        String serviceToken = headers.getValue(SERVICE_AUTHORIZATION);
+        UserInfo userInfo = authorizationHeadersProvider.getUserInfo(userToken);
+
+        Document document = documentManagementFiles.getDocument(NOTICE_OF_APPEAL_PDF);
+
+        StartEventResponse startCase = coreCaseDataApi.startForCaseworker(
+            userToken,
+            serviceToken,
+            userInfo.getUid(),
+            "IA",
+            "Asylum",
+            "startAppeal"
+        );
+
+        String resourceFilename = "requests/ccd/case_data.json";
+
+        Map data = null;
+        try {
+            String caseDataString =
+                FileUtils.readFileToString(ResourceUtils.getFile("classpath:" + resourceFilename), "UTF-8");
+            caseDataString = caseDataString.replace(
+                "{NOTICE_OF_DECISION_DOCUMENT_STORE_URL}",
+                document.getDocumentUrl()
+            );
+            caseDataString = caseDataString.replace(
+                "{NOTICE_OF_DECISION_DOCUMENT_NAME}",
+                document.getDocumentFilename()
+            );
+            caseDataString = caseDataString.replace(
+                "{NOTICE_OF_DECISION_DOCUMENT_STORE_URL_BINARY}",
+                document.getDocumentBinaryUrl()
+            );
+
+            data = new ObjectMapper().readValue(caseDataString, Map.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        CaseDataContent caseDataContent = CaseDataContent.builder()
+            .eventToken(startCase.getToken())
+            .event(Event.builder()
+                .id(startCase.getEventId())
+                .summary("summary")
+                .description("description")
+                .build())
+            .data(data)
+            .build();
+
+        //Fire submit event
+        CaseDetails caseDetails = coreCaseDataApi.submitForCaseworker(
+            userToken,
+            serviceToken,
+            userInfo.getUid(),
+            "IA",
+            "Asylum",
+            true,
+            caseDataContent
+        );
+
+        log.info("Created case [" + caseDetails.getId() + "]");
+
+        StartEventResponse submitCase = coreCaseDataApi.startEventForCaseWorker(
```

Comment:
Pact test for startEventForCaseWorker. See https://github.com/hmcts/wa-task-management-api/pull/249/files as you can reuse these.